planning/behavior_path_planner/src/utils/start_planner/geometric_pull_out.cpp

```diff
@@ -80,15 +80,48 @@ boost::optional<PullOutPath> GeometricPullOut::plan(Pose start_pose, Pose goal_p
         parameters_.collision_check_margin)) {
     return {};
   }
+  const double velocity = parallel_parking_parameters_.forward_parking_velocity;
 
   if (parameters_.divide_pull_out_path) {
     output.partial_paths = planner_.getPaths();
+    /*
+    Calculate the acceleration required to reach the forward parking velocity at the center of
+    the front path, assuming constant acceleration and deceleration.
+    v                     v
+    |                     |
+    |    /\               |    /\
+    |   /  \              |   /  \
+    |  /    \             |  /    \
+    | /      \            | /      \
+    |/________\_____ x    |/________\______ t
+    0  a_l/2  a_l         0    t_c 2*t_c
+    Notes:
+    a_l represents "arc_length_on_path_front"
+    t_c represents "time_to_center"
+    */
     // insert stop velocity to first arc path end
     output.partial_paths.front().points.back().point.longitudinal_velocity_mps = 0.0;
+    const double arc_length_on_path_front =
+      motion_utils::calcArcLength(output.partial_paths.front().points);
+    const double time_to_center = arc_length_on_path_front / (2 * velocity);
+    const double average_velocity = arc_length_on_path_front / (time_to_center * 2);
+    const double average_acceleration = average_velocity / (time_to_center * 2);
+    output.pairs_terminal_velocity_and_accel.push_back(
+      std::make_pair(average_velocity, average_acceleration));
+    const double arc_length_on_path_back =
+      motion_utils::calcArcLength(output.partial_paths.back().points);
```

Comment:
`output.partial_paths.back().points` is not up to end pose but combined with road reference path

We need to calculate arc length up to PullOutPath.end_pose 

https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/include/behavior_path_planner/utils/start_planner/pull_out_path.hpp#L31

```
struct PullOutPath
{
  std::vector<PathWithLaneId> partial_paths{};
  Pose start_pose{};
  Pose end_pose{};
};
```

Reply:
@kosuke55 
fix with the [commit](https://github.com/autowarefoundation/autoware.universe/pull/4669/commits/e566b3d099faf7704f416dca895561e8e7f8f7cc)

Reply:
finally...
https://github.com/autowarefoundation/autoware.universe/pull/4669/commits/81d756d7ef2732e748feef70bf59b82e6b0a7841